apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -83,6 +83,8 @@ export interface AichatState {
   studentChatHistory: ChatEvent[];
   // Denotes whether we should show the warning modal
   showWarningModal: boolean;
+  // Denotes whether we should show the teacher onboarding modal
+  showOnboardingModal: boolean;
```

Comment:
Yep that's right, since the initial modal state is dependent on a redux property (isUserTeacher), we can't and shouldn't access it in the redux file directly. Setting it in the component is the right thing to do.